### PR TITLE
Override cw default compile costs

### PR DESCRIPTION
**Pull request by the-frey**
Addresses #85 

I've now checked that the node runs at least but will run some more checks over the weekend.

The bash scripts I'm now using to bootstrap the app in dev are pretty useful so I've committed and documented them because tbh people have been asking on discord & it's on my to-do list to improve the dev section of the docs.

Probably could use a once-over for sanity-checking before I spam a bunch of contracts at it.

Once we're happy with code integrity here and want to merge, there's the question of what to actually change the value to. I assume we'll want to raise it a bit more than what I've done here and run some tests on uni? @JakeHartnell - thoughts?

### CODE REVIEW

**Review by JakeHartnell — APPROVED** [review]
Yolo.

**Comment by JakeHartnell** [comment]
Build and Deploy is failing because of this dumb file: https://github.com/CosmosContracts/juno/blob/main/.github/workflows/pages.yml

I vote we remove it as we're not really using the Starport vue stuff. :shrug:

File: app/wasm_config.go [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+	return wasmkeeper.WasmGasRegisterConfig{
+		InstanceCost:               wasmkeeper.DefaultInstanceCost,
+		CompileCost:                DefaultJunoCompileCost,
+		GasMultiplier:              wasmkeeper.DefaultGasMultiplier,
+		EventPerAttributeCost:      wasmkeeper.DefaultPerAttributeCost,
+		CustomEventCost:            wasmkeeper.DefaultPerCustomEventCost,
+		EventAttributeDataCost:     wasmkeeper.DefaultEventAttributeDataCost,
+		EventAttributeDataFreeTier: wasmkeeper.DefaultEventAttributeDataFreeTier,
+		ContractMessageDataCost:    wasmkeeper.DefaultContractMessageDataCost,
+	}
```
Comment (giansalex) [inline app/wasm_config.go:22]: maybe we don't need to modify this, if new parameters will be added to `WasmGasRegisterConfig`
```suggestion
	gasConfig := wasmkeeper.DefaultGasRegisterConfig()
	gasConfig.CompileCost = DefaultJunoCompileCost

	return gasConfig
```

File: app/wasm_config.go [hunk 2]
```diff
@@ -0,0 +1,27 @@
+package app
+
+import (
+	wasmkeeper "github.com/CosmWasm/wasmd/x/wasm/keeper"
+)
+
+const (
+	DefaultJunoCompileCost uint64 = 4
```
Comment (JakeHartnell) [inline app/wasm_config.go:8]: Can we YOLO make this number 10.

Comment (JakeHartnell) [reply app/wasm_config.go:8]: :joy_cat:

Comment (JakeHartnell) [inline app/wasm_config.go:8]:
```suggestion
	DefaultJunoCompileCost uint64 = 10
```

**Comment by faddat** [comment]
Agree about YOLO make it ten.

expensive is **good**.

**Comment by the-frey** [comment]
I think as mentioned on TG that if this were tagged correctly, I can use this as the target binary for the instructions for the `moneta-beta` upgrade handler, meaning it doesn't need its own upgrade handler. Otherwise would maybe need one as iirc these param changes are consensus-breaking.

**Comment by the-frey** [comment]
@faddat / @giansalex happy for these changes to go in?

**Comment by giansalex** [comment]
@the-frey yes, must be included in moneta-beta upgrade

**Comment by dimiandre** [comment]
Looks good to me! 

Can these params be modified by governance?

**Review by dimiandre — APPROVED** [review]

**Review by JakeHartnell — APPROVED** [review]

**Comment by the-frey** [comment]
@dimiandre I think we could make them modifiable by governance in future, but for the moment they have to be changed in our codebase.

**Pull request merged**